src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -170,6 +137,71 @@ const ProjectExternalSubmissionsTable = ({
     beforeFormComponent: valuesViewComponent,
   });
 
+  const getColumnDefs = useCallback(() => {
+    return [
+      {
+        header: 'ID',
+        render: (s: ExternalFormSubmissionSummaryFragment) => s.id,
+      },
+      {
+        header: 'Status',
+        linkTreatment: false,
+        render: (s: ExternalFormSubmissionSummaryFragment) => {
+          const isNew = s.status === ExternalFormSubmissionStatus.New;
+          return (
+            <>
+              <Chip
+                label={capitalize(s.status)}
+                size='small'
+                color={isNew ? 'primary' : 'default'}
+                variant={isNew ? 'filled' : 'outlined'}
+                sx={isNew ? {} : { color: theme.palette.text.secondary }}
+              />
+              {s.spam && (
+                <Chip
+                  label='Spam'
+                  size='small'
+                  color='error'
+                  variant='outlined'
+                  sx={{ ml: 1, color: theme.palette.error.dark }}
+                />
+              )}
+            </>
+          );
+        },
+      },
+      {
+        header: 'Date Submitted',
```

Comment:
Should we update the "Date Submitted" column title now that it's a relative time?

Reply:
We could use this existing component which displays both time and relative time, and keep the label the same
```
<RelativeDateTableCellContents
            dateTimeString={s.submittedAt}
            horizontal
          />
```

Reply:
great!